lib/ansible/cli/galaxy.py

```diff
@@ -71,15 +72,15 @@
     ('password', False, 'str'),
     ('token', False, 'str'),
     ('auth_url', False, 'str'),
-    ('v3', False, 'bool'),
+    ('api_version', False, 'int'),
     ('validate_certs', False, 'bool'),
     ('client_id', False, 'str'),
     ('timeout', False, 'int'),
 ]
 
 # config definition fields
 SERVER_ADDITIONAL = {
-    'v3': {'default': 'False'},
```

Comment:
I don't think this was ever documented, but since there was no internal-only warning, is worth mentioning in a changelog?

Reply:
Done